src/config/feature-flags.ts

```diff
@@ -29,6 +29,7 @@ export enum BooleanFlags {
 	ENABLE_GENERIC_CONTAINERS = "enable-generic-containers",
 	ENABLE_GITHUB_SECURITY_IN_JIRA = "enable-github-security-in-jira",
 	DELETE_MESSAGE_ON_BACKFILL_WHEN_OTHERS_WORKING_ON_IT = "delete-message-on-backfill-when-others-working-on-it",
+	TEMP_LOGS_FOR_DOS_TICKETS = "temp-logs-for-dos-tickets",
```

Comment:
Can you use `VERBOSE_LOGGING` instead?

Reply:
Or does it make sense to use `VERBOSE_LOGGING`?

Reply:
I can. Just wondering if it's going to add any extra value though?

Reply:
Actually, I just checked that feature flag and it looks like it's already on for several customers. I really want to limit this to the 2 customers who I'm helping out. Starting with one that has given permission for me to log this additional data (waiting for the other one to get back to me).